### ISSUE TITLE
Use envrionment instead of config

### ISSUE THREAD

**Issue by sgkim126** [CodeChain-io/cron-jobs#61]
Currently, there are two ways to set up the configuration, because each tool is written by the various developers.
One is using `config` and the other is using environments.
Some developers, including me, prefer `config` to environments because the environments vanish when the session closes.
And others prefer environments because it corrupts the git history and it's still hard to track because it depends on environments to choose the configuration file.

But now, we can keep the environment because we use tmuxinator, so I think it's better to unify styles by using environment variables.

**Comment by sgkim126** [comment]
@HoOngEe @majecty @foriequal0 What do you think about it?

**Comment by majecty** [comment]
I agree to unify them. Both `config` and ENV variable have some inconvenience, but I agree to unify it with the Environment variable.

**Comment by foriequal0** [comment]
Do you mean that `config` is using JSON file or a ts file like tick-tock, and using environment variables is like fee-monitor?
I was intended to use an environment variable to switch some targets (dev, corgi, mainnet) or some temporary fields. It was complementary to the config file.

**Comment by sgkim126** [comment]
@foriequal0 Using the chain name is not flexible to run various testnet. Please change it to use the environment to set the variable it needed instead of hard-coding it.

**Comment by foriequal0** [comment]
I don't think we need that much flexibility. Selecting between presets like `codechain --chain solo` would be enough to us.

**Comment by sgkim126** [comment]
@foriequal0 FYI, --chain option can set the file, you can run any chain not only the pre-defined one.
And https://github.com/CodeChain-io/cron-jobs/pull/60 it's the one example of the inflexibility of fee-monitor. We can add more testnet to test some features and you should modify code every time you add the testnet.

**Comment by foriequal0** [comment]
Okay. Should we make them designate config file? I think we need to make the config overlappable like ~/.gitconfig to store sensitive data.

**Comment by sgkim126** [comment]
@foriequal0 Why not using environment variables?
